frontend/pages/hosts/ManageHostsPage/components/RunScriptBatchModal/RunScriptBatchModal.tsx

```diff
@@ -91,8 +161,12 @@ const RunScriptBatchModal = ({
             runByFilters
               ? totalFilteredHostsCount.toLocaleString()
               : selectedHostIds.length.toLocaleString()
-          } hosts, or will run as each host comes online. See host details for individual results.`
+          } host${selectedHostIds.length === 1 ? "" : "s"}, or will run ${
+            selectedHostIds.length === 1 ? "when the" : "as each"
+          } host comes online. See host details for individual results.`
         );
+        onCancel();
+        // TODO -- redirect to the batch scripts page.
```

Comment:
TODO once that page is done, or now?

Reply:
i was thinking we'd do it once the page was done, but I can update it now if you prefer